src/proto/BitcoinGold.proto

```diff
@@ -0,0 +1,12 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+syntax = "proto3";
+
+package TW.BitcoinGold.Proto;
+option java_package = "wallet.core.jni.proto";
+
+import "Bitcoin.proto";
```

Comment:
`./tools/generate-files`:

BitcoinGold.proto:12:1: warning: Import Bitcoin.proto but not used.